src/main/webapp/templates/builders/dialogBuilder.js

```diff
@@ -32,12 +32,27 @@ export default class DialogBuilder {
     this.isLoading = value;
   }
 
+  removeJiraMentionMarkdown(message) {
+    const mainPatt = new RegExp('(<mention email=\\")|(\\"\\/>)');
```

Comment:
This method can be a single replace, like:
message.replace('<mention email="(.*)"\/>', '$1')